docs/examples.html

```diff
@@ -33,7 +33,7 @@
 
       <pre class="code" data-fill-from=".js-code-basic"></pre>
 
-<script type="text/x-example-code" class="js-code-basic">
+      <script type="text/x-example-code" class="js-code-basic">
```

Comment:
The code blocks aren't indented because the indentation is copied over to how it is displayed. So if the code is indented in slightly in the markup, the example code in the docs will be also.

It tends to look out of place: http://select2.github.io/select2/#disabled-options